src/test/java/uk/gov/ons/census/casesvc/service/CaseServiceTest.java

```diff
@@ -48,8 +48,8 @@
       new byte[] {0x10, 0x20, 0x10, 0x20, 0x10, 0x20, 0x10, 0x20};
   private static final Integer CE_ACTUAL_CAPACITY = 0;
   private static final String TEST_ADDRESS_TYPE_CE = "CE";
-  private List<String> directDeliveryTreatmentCodes =
-      new ArrayList<>(Arrays.asList("CE_LDIEE", "test"));
+  private static final Set<String> DIRECT_DELIVERY_TREATMENT_CODE =
```

Comment:
This is unnecessary if we say that the treatment codes are an implicit part of the CaseService class... not config